frontend/src/App.jsx

```diff
@@ -0,0 +1,13 @@
+import './App.css'
+import MapComponent from './components/MapComponent'
+function App() {
+
+  return (
+    <div className="App">
+    <h1>Maps Application</h1>
+    <MapComponent />
+  </div>
```

Comment:
Given your use of material-ui, it's preferable to employ controlled components like Box and Typography, rather than relying solely on the vanilla div and h1 elements